### ISSUE TITLE
How to set tokens for in memory users

### ISSUE THREAD

**Issue by petrovic-ivan** [IdentityServer/IdentityServer3#3381]
### Question 
Hello, 

Is it possible bind already generated token to In Memory user? I don't want to add as claims, I want to be able to call RequestAuthorizationCodeAsync() and return TokenResponse from my application.

**Comment by leastprivilege** [comment]
I don't follow.

**Comment by petrovic-ivan** [comment]
Basically I want to configure couple in memory users in IdentityServer3. I did that in IdentityServerServiceFactory.UseInMemoryUsers() method. Now I have confugured users and it works properly. Next thing what I like (if is possible) is to "configure" IdentityServer3 to return tokens for specific user when I am calling method RequestAuthorizationCodeAsync() in my application. That method returns TokenResponse but with generated Token (I guess). I would like to manually add already generated tokens somehow to IdentityServer3 and bind to in memory users. Is it possible to do that?
